mesonbuild/compilers.py

```diff
@@ -204,16 +204,16 @@ def is_library(fname):
                 'b_coverage': coredata.UserBooleanOption('b_coverage',
                                                          'Enable coverage tracking.',
                                                          False),
-                'b_colorout' : coredata.UserComboOption('b_colorout', 'Use colored output',
-                                                        ['auto', 'always', 'never'],
-                                                        'always'),
-                'b_ndebug' : coredata.UserBooleanOption('b_ndebug',
-                                                        'Disable asserts',
-                                                        False),
-                'b_staticpic' : coredata.UserBooleanOption('b_staticpic',
-                                                           'Build static libraries as position independent',
-                                                           True),
-                }
+                'b_colorout': coredata.UserComboOption('b_colorout', 'Use colored output',
+                                                       ['auto', 'always', 'never'],
+                                                       'always'),
+                'b_ndebug': coredata.UserBooleanOption('b_ndebug',
+                                                       'Disable asserts',
+                                                       False),
+                'b_staticpic': coredata.UserBooleanOption('b_staticpic',
+                                                          'Build static libraries as position independent',
+                                                          True),
+               }
```

Comment:
__[flake8]__
* [E126] continuation line over-indented for hanging indent

<!-- Comment by [SideCI](https://sideci.com) -->